src/app/views/settings/Settings.tsx

```diff
@@ -82,26 +82,29 @@ function Settings(props: ISettingsProps) {
       }
     ];
 
+    if (permissionModeType === PERMISSION_MODE_TYPE.User) {
+      menuItems.push({
+        key: 'view-all-permissions',
+        text: translateMessage('view all permissions'),
+        iconProps: {
+          iconName: 'AzureKeyVault',
+        },
+        onClick: () => changePanelState(),
+      });
+    }
+
```

Comment:
does this mess up the order of the menuItems for User mode? since you're pushing `View all permissions` here instead of in the `if (authenticated)` if statement.

Reply:
yeah it does, should i do it so the order is not changed? but then there's three if statements 

like 
if authed -> use explorer as (app/user)
if perimissionModeType is user -> select permissions
if autthed -> signout

Reply:
Anne said the order is ok!